examples/ERP/plot_embedding_MEG.py

```diff
@@ -78,3 +82,26 @@
 ax.grid(False)
 ax.legend()
 plt.show()
+
+###############################################################################
+# with Riemannian Locally Linear Embedding (RLLE)
+
+rlle = RiemannLLE(n_components=2)
+embd = rlle.fit_transform(covs)
+
+###############################################################################
+# Plot the two first components of the RLLE embedded points
+
+fig, ax = plt.subplots(figsize=(7, 8), facecolor='white')
```

Comment:
That function is unfortunately assuming rescaled embedding values between -1 and 1, which I don't provide and which would break the transform function for new data points.